local/local_provider.go

```diff
@@ -63,10 +70,86 @@ func (r *Local) CreateDatabase(db *crd.Database) (string, error) {
 	return db.Name, nil
 }
 
+const (
+	defaultLocalRDSPVSize = "1Gi"
```

Comment:
I don't like default values... could we not just fail the upload of the YAML file if it's not configured